src/main/java/Duke.java

```diff
@@ -20,62 +37,64 @@ public static void printMessage(String message) {
-        Scanner in = new Scanner(System.in);
-        TaskManager taskManager = new TaskManager();
+        printStartingOrEndingMessage(IS_STARTING);
+        runDuke();
+        printStartingOrEndingMessage(IS_ENDING);
+    }
 
+    private static void runDuke() {
         boolean conversationIsOver = false;
         while (!conversationIsOver) {
-            String inputCommand = in.nextLine().trim();
-            //switch to lowercase so that Duke won't be case-sensitive
-            String command = inputCommand.toLowerCase().split(" ")[0];
+            String inputCommand = SCANNER.nextLine().trim();
+            String command = getFirstWord(inputCommand);
             switch (command) {
             case "list":
-                taskManager.printTasks();
+                TASK_MANAGER.printTasks();
                 break;
             case "done":
-                taskManager.setTaskAsDone(Integer.parseInt(inputCommand.split(" ")[1]));
+                int indexOfTaskDone = getIntegerFromCommand(inputCommand);
+                TASK_MANAGER.setTaskAsDone(indexOfTaskDone);
                 break;
             case "bye":
                 conversationIsOver = true;
                 break;
             case "deadline":
-                //to remove the word "deadline"
-                String deadlineInput = inputCommand.substring(8).trim();
-                taskManager.addDeadline(deadlineInput);
+                String deadlineInput = removeFirstWordInSentence(inputCommand, 8);
+                TASK_MANAGER.addDeadline(deadlineInput);
                 break;
             case "todo":
-                //to remove the word "todo"
-                String todoInput = inputCommand.substring(4).trim();
-                taskManager.addTodo(todoInput);
+                String todoInput = removeFirstWordInSentence(inputCommand, 4);
+                TASK_MANAGER.addTodo(todoInput);
                 break;
             case "event":
-                //to remove the word "event"
-                String eventInput = inputCommand.substring(5).trim();
-                taskManager.addEvent(eventInput);
+                String eventInput = removeFirstWordInSentence(inputCommand, 5);
+                TASK_MANAGER.addEvent(eventInput);
                 break;
             default:
                 printMessage(HELP_MESSAGE);
                 break;
             }
         }
-        printStartingOrEndingMessage(false);
+    }
+
+    private static String removeFirstWordInSentence(String inputCommand, int i) {
+        //to remove the words "deadline", "even" or "todo"
+        return inputCommand.substring(i).trim();
+    }
+
+    private static int getIntegerFromCommand(String input) {
+        return Integer.parseInt(input.split(" ")[1]);
+    }
+
+    private static String getFirstWord(String inputCommand) {
+        //switch to lowercase so that Duke won't be case-sensitive
+        return inputCommand.toLowerCase().split(" ")[0];
     }
```

Comment:
Suggestion:
Add comments to what some of these method does including their @param and @return.